lib/Dialect/AIEVec/Transforms/VectorToAIEVecConversions.cpp

```diff
@@ -2637,14 +2676,17 @@ static void populateAIEVecV1ConversionPatterns(RewritePatternSet &patterns,
 
 static void populateAIEVecV2ConversionPatterns(RewritePatternSet &patterns,
                                                TargetBackend backend) {
-  if (backend == TargetBackend::CPP) {
-    patterns.add<LowerVectorTransferReadToAIEUPD>(patterns.getContext(), 128,
-                                                  1024, 256, 1024);
-  }
   // clang-format off
   // TODO: Reorder these alphabetically
+  if (backend == TargetBackend::CPP) {
+    patterns.add<
+        LowerVectorTransferReadToAIEUPD
+      >(patterns.getContext(), 128, 1024, 256, 1024);
+    patterns.add<
+        LowerVectorAddIOpToAIEVecAddElemOp
```

Comment:
We should move `LowerVectorSubIOpToAIEVecSubElemOp` to here as well.

Reply:
Aren't those applicable for LLVMIR conversion as well?